.tours/welcome-to-dreamhouse.tour

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
```

Comment:
What's this schema? I haven't seen it in other code tours

Reply:
I guess that it's used for JSON validation, this got added automatically when I created the tour. I suggest keeping it since it belongs the extension. The reason why we don't have it is that there were a couple of extension updates since we created the other tours.